sdk/eventhub/event-hubs/test/receiver.spec.ts

```diff
@@ -134,7 +133,11 @@ describe("EventHub Receiver #RunnableInBrowser", function(): void {
       const partitionId = partitionIds[0];
       const pInfo = await client.getPartitionInformation(partitionId);
       debug(`Creating new receiver with last enqueued offset: "${pInfo.lastEnqueuedOffset}".`);
-
+      breceiver = BatchingReceiver.create(
+        (client as any)._context,
+        parseInt(partitionId),
+        EventPosition.fromOffset(pInfo.lastEnqueuedOffset)
```

Comment:
`breceiver` is not used in this test. This might have gotten added when resolving merge conflicts